utility/mirrormanager2.spec

```diff
@@ -124,6 +124,16 @@ Requires(pre):  shadow-utils
 Install a number of utility scripts to be used manually or in cron jobs to
 run MirrorManager.
 
+%package client
+Summary:        Fedora mirror management system downstream mirror tools
+Group:          Applications/Internet
```

Comment:
thinking more about it, might be still needed if we want to target EL5. I don't know for the client but I doubt MM2 will run on EL5.